source/_docs/ecosystem/nginx.markdown

```diff
@@ -111,6 +111,7 @@ http {
 
         # These shouldn't need to be changed
         listen 443 default_server;
+        listen [::]:443 default_server ipv6only=on;
```

Comment:
Unless you are on a very old machine you should be able to just
```
listen [::]:443 ssl http2 default_server;
```

As `sysctl net.ipv6.bindv6only` defaults to `0` and so binding to ipv6 will also bind to the ipv6-mapped ipv4 address.